xbmc/GUIInfoManager.cpp

```diff
@@ -1000,6 +1000,11 @@ const infomap weather[] =        {{ "isfetched",        WEATHER_IS_FETCHED },
 ///     @return **True** if there is a CD or DVD in the DVD-ROM drive.
 ///     <p>
 ///   }
+///   \table_row3{   <b>'System.HasMediaAudioCD'</b>,
+///                  \anchor System_HasMediaAudioCD
+///                  _boolean_
+///     @return **True** if there is an audio CD in the optical drive. Will return **False** in case of a DVD or a BR.
```

Comment:
* "BR" => "BD"
* It will also return false if system has no optical drive ;-)
* if this is for Leia, add
```
///     <p><hr>
///     @skinning_v18 **[New Boolean Condition]** \link System_HasMediaAudioCD `System.HasMediaAudioCD`\endlink
///     <p>
```